build/ci/check_demo_pr_checks.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -xe
+set +o pipefail
+
+export passed_k8s_checks=true
+export passed_openshift_checks=true
+
+wait_for_checks_to_complete(){
+  community_operators_branch=$1
+  all_checks_passed=true
+  export repo_pr=`gh pr list --repo $github_csiblock_community_operators_repository | grep $community_operators_branch`
+  if [[ "$repo_pr" == *"$community_operators_branch"* ]]; then
+    sleep 5
+    while [ "$(gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository | grep -i pending | wc -l)" -gt 0 ]; do
+      sleep 15
+    done
+    summary_counter=0
+    export search_summary_test=`gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository | grep -i summary`
```

Comment:
can't this be initialized with a simpler, constant value?

Reply:
I don't think so, its the only way we can get the status of checks

Reply:
I meant that it's already being executed [inside the `while`](https://github.com/IBM/ibm-block-csi-operator/blob/f58e6f493eeba993456e6786e97eb3cbb6c08c78/build/ci/check_demo_pr_checks.sh#L20), so I don't think we need to execute it here